apps/music_proxy.cpp

```diff
@@ -178,6 +178,10 @@ class SteeringHandler
 
     SteeringHandler( MUSIC::Setup* setup, const std::string& steeringPort )
         : _state( monsteer::steering::SimulationPlaybackState::PLAY )
+        , _stimuliInjectionEvent( monsteer::steering::EVENT_STIMULUSINJECTION,
+                                  boost::bind( &SteeringHandler::_onStimulusInjection, this, _1 ))
+        , _playbackStateEvent( monsteer::steering::EVENT_PLAYBACKSTATE,
+                               boost::bind( &SteeringHandler::_onPlaybackStateChange, this, _1 ))
```

Comment:
I can use std::bind I think ( the are compatible with std::bind I suppose ). std::bind is also C++11. Do we have a limitation on C++11 in monsteer ?